README.md

```diff
@@ -292,7 +292,7 @@ Usage of ./sriovdp:
 
 ### Assumptions
 
-This plugin does not bind or unbind any driver to any device whether it's PFs or VFs. It also doesn't create Virtual functions either. Usually, the virtual functions are created at boot time when kernel module for the device is loaded. Required device drivers could be loaded on system boot-up time by white-listing/black-listing the right modules. But plugin needs to be aware of the driver type of the resources (i.e. devices) that it is registering as K8s extended resource so that it's able to create appropriate Device Specs for the requested resource.
+This plugin does not bind or unbind any driver to any device whether it's PFs or VFs. It also doesn't create Virtual functions either. Usually, the virtual functions are created at boot time when kernel module for the device is loaded. Required device drivers could be loaded on system boot-up time by allow-listing/deny-listing the right modules. But plugin needs to be aware of the driver type of the resources (i.e. devices) that it is registering as K8s extended resource so that it's able to create appropriate Device Specs for the requested resource.
```

Comment:
> allow-listing/deny-listing

thats a new terminology introduced to linux kernel ?
can you provide some background on this initiative ?

what i know is that `blacklist` term is used when one wants to prevent kernel from loading a module.
e.g add `blacklist <driver-name>` line in `/etc/modporbe.d/blacklist.conf`

Reply:
> thats a new terminology introduced to linux kernel 
Yes but only for new features in linux kernel. Previous features using this terminology will not be updated.
https://elixir.bootlin.com/linux/v5.11-rc2/source/Documentation/process/coding-style.rst#L322

> can you provide some background on this initiative ?

https://lwn.net/Articles/825319/

> what I know is that blacklist term is used when one wants to prevent kernel from loading a module.
e.g add blacklist <driver-name> line in /etc/modporbe.d/blacklist.conf

Yes and this terminology will not change for existing features but new features in Linux kernel will replace this terminology.

Reply:
Thanks Martin.

Even though on existing OSes the old terminology is still used, i think that the intention of the new terminology is clear.